eng/tox/run_pylint.py

```diff
@@ -59,21 +59,22 @@
             exit(0)
 
     try:
-        check_call(
-            [
+        args = [
                 sys.executable,
                 "-m",
                 "pylint",
                 "--rcfile={}".format(rcFileLocation),
                 "--output-format=parseable",
+                "--py-version=3.8",
```

Comment:
What's the primary difference between setting `basepython` in the tox file, and setting `--py-version` when running pylint?

The latter does make sense, but did the former not work?

Reply:
py-version sets the minimum python version to run version dependent checks on

setting basepython required users to have python38 installed which was pointed out as a pain point

I am currently investigating this switch over though because it does seem to cause different pylint behavior